src/paperoni/__main__.py

```diff
@@ -126,7 +126,8 @@ class Refine:
     # [action: append]
     link: list[str]
 
-    # Tags to refine
+    # Refine tags. Refine will be executed on all strategies tagged with at
+    # least one of the tags. Use "all" to refine using all strategies.
```

Comment:
Sorry, I've approved the changes related to tags in you last PR but the tag system doesn't behave like I was expecting. Let me know if I'm mistaking with this written behaviour

Reply:
Currently it's strategies tagged with all of the tags, not at least one (except for the "all" tag, which uses all strategies).